src/schema/hotel-price-crawling.json

```diff
@@ -129,6 +129,46 @@
                 },
                 "canBeShared": {
                     "type": "boolean"
+                },
+                "bed": {
+                    "type": "array",
+                    "description": "Types of the beds.",
+                    "minItems": 0,
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "single",
+                            "double",
+                            "semi-double",
+                            "queen",
+                            "king",
+                            "twin",
+                            "3",
+                            "4",
+                            "more than 4"
+                        ]
+                    }
+                },
+                "style": {
+                    "type": "string",
+                    "description": "Style of room for japanese hotels.",
+                    "enum": [
+                        "japanese",
+                        "western"
+                    ]
+                },
+                "features": {
+                    "type": "array",
+                    "description": "Room characteristics",
+                    "minItems": 0,
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "smoking",
+                            "non-smoking",
+                            "openAirBath"
```

Comment:
This is a bit strange. Do we need to mix `foo-bar` and `fooBar` styles here? Probably it should be `open-air-bath` like in all other enums.